core/src/main/scala/kafka/zk/ZkData.scala

```diff
@@ -58,6 +58,30 @@ object BrokerIdsZNode {
   def encode: Array[Byte] = null
 }
 
+class BrokerInfo(id: Int,
+                 host: String,
+                 port: Int,
+                 advertisedEndpoints: Seq[EndPoint],
+                 jmxPort: Int,
+                 rack: Option[String],
+                 apiVersion: ApiVersion) {
+
+  def ID(): Int = {
+    id
+  }
+
+  def path(): String = {
+    BrokerIdZNode.path(id)
+  }
+
+  def endpoints(): String = {
+    advertisedEndpoints.mkString(",")
+  }
+  def getBytes(): Array[Byte] = {
```

Comment:
Add a new line above. Also, it's better to name the method encode().

Reply:
done